### ISSUE TITLE
Split talk messages are shown backwards in stream

### ISSUE THREAD

**Issue by ohAitch** [urbit/talk#26]
Reversing the talk stream also reverses any auto-split messages in that stream, "long talk message" becoming

```
message
talk
long
```

Fixing this would probably require backend support for threading/grouping of split messages.

**Comment by ohAitch** [comment]
Fix proposed by @cgyarvin: flip the displayed per-speaker clusters of messages right-side-down, regardless of whether they were the products of a split or not. This could run into problems when dozens of messages are coming from the same person, but indeed shouldn't be difficult to implement.

**Comment by galenwp** [comment]
I sort of think stream should just scroll up.

**Comment by galenwp** [comment]
You are talking about `urbit.org/stream`, right?

**Comment by Fang-** [comment]
I don't think the issue from the OP occurs anymore: groups of messages by the same author flow chronologically top-to-bottom... while those message groups flow bottom-to-top.

>I sort of think stream should just scroll up.

Agreed, this would bring it in line with what "real" webtalk does, and is also more logical and easy to read.  
However, there's content above the talk module on the stream page. Page content and (recent) talk content become separated, and scrolling up to reach page content causes the talk module to load more messages, pushing you down again.

Placing the module at the top of `stream.md` still makes the navigation header unreachables due to message scrollback, but rendering the module above the header would require a special case, which is probably not desired. (And it'd still look kinda off anyway.)

Though slightly inconsistent, the easiest solution would be to have the stream page use the sidebar navigation instead of the header one.  
Alternatively, the messages could be wrapped in their own scrollable element with a set height.

tl;dr: One does not simply make stream scroll up. Opinions, suggestions? I'd be willing to put in some work.
